### ISSUE TITLE
can’t find the file

### ISSUE THREAD

**Issue by 2021270902001sc** [dingdingcai/GSPose#7]
I have downloaded part of the megapose-gso-1M dataset. And the python dataset/extract_megapose_to_BOP.py was executed, but I couldn't find valid_meshes.json in MegaPose.
Code:
training/training.py:50-57
DATA_DIR = os.path.join(PROJ_ROOT, 'dataspace', 'MegaPose')

dataset = Dataset(data_dir=DATA_DIR,
                  query_view_num=que_view_num,
                  refer_view_num=refer_view_num,
                  rand_view_num=random_view_num,
                  nnb_Rmat_threshold=nnb_Rmat_threshold, 
                 )
dataset/megapose_dataset.py:81-85

        cad_model_dir = os.path.join(self.data_dir, 'models')
        valid_mesh_names = mmcv.load(os.path.join(self.data_dir, 'valid_meshes.json'))
        valid_mesh_info_path = os.path.join(self.data_dir, 'valid_meshes_diameters.json')
        valid_mesh_bbox3D_path = os.path.join(self.data_dir, 'valid_meshes_3D_bboxes.json')

valid_mesh_names Is dataspace/MegaPose/valid_meshes.json，But I can't find this file.

**Comment by binbin0801** [comment]
Have you solved this problem?

**Comment by 2021270902001sc** [comment]
No,I find dataspace/MegaPose/google_scanned_objects/valid_meshes.json, but google_scanned_objects doesn't have cad_model_dir(models)
The train_pbr which I have doesn't have any .json,I guess the .json is created by other files.

**Comment by 7-Ryan** [comment]
Have you solved the problem? Can you complete the code train `python training/training.py`?
